lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -1999,9 +2000,10 @@ internal class EvaluatingCompiler(
     }
 
     private fun compileTime(node: DateTimeType.Time) : ThunkEnv {
-        val (hour, minute, second, nano, precision, tz_minutes, metas) = node
+        val (hour, minute, second, nano, precision, with_time_zone, tz_minutes, metas) = node
         return thunkFactory.thunkEnv(metas) {
-            valueFactory.newTime(Time.of(hour, minute, second, nano, precision, tz_minutes))
+            // Add the default time zone if the type "TIME WITH TIME ZONE" does not have an explicitly specified time zone.
+            valueFactory.newTime(Time.of(hour, minute, second, nano, precision, if (with_time_zone && tz_minutes == null) DEFAULT_TIMEZONE_OFFSET.totalMinutes else tz_minutes))
```

Comment:
> Consider putting arguments on separate lines.

Is this fine? 

<img width="870" alt="Screen Shot 2021-09-23 at 3 02 49 PM" src="https://user-images.githubusercontent.com/67429381/134590191-28695f87-0bfd-4917-ac44-2c0827e81883.png">

Reply:
Or this one? 

<img width="1061" alt="Screen Shot 2021-09-23 at 3 05 05 PM" src="https://user-images.githubusercontent.com/67429381/134590402-ceef18cf-5f6e-4ba2-89fa-e2e98b9ffc1e.png">

Reply:
As an aside, GitHub allows you to use code blocks which support syntax highlighting (article on [code blocks](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)). This can be easier to create and read than screenshots.